src/safe_apps/views.py

```diff
@@ -29,9 +29,21 @@ class SafeAppsListView(ListAPIView):
         description="Used to filter Safe Apps that are available on `clientUrl`",
         type=openapi.TYPE_STRING,
     )
+    _swagger_url_param = openapi.Parameter(
+        "url",
+        openapi.IN_QUERY,
+        description="Filter Safe Apps available from `url`",
```

Comment:
nit: would include in the description that the filter looks for an exact match

Reply:
Yeah that makes sense! Will update it.

Reply:
Updated here: [c983aad](https://github.com/safe-global/safe-config-service/pull/532/commits/c983aad10be6553c85cd72a29a4a0a85967fb2bc)

<img width="651" alt="image" src="https://user-images.githubusercontent.com/3332770/169004949-9d415943-640c-4cf0-92ab-f15fed81c8fe.png">